packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -1716,6 +1729,33 @@ export default class SchedulerTask {
                 details: { error: e.message },
             });
 
+            if (
```

Comment:
We should capture this with a typed error rather than parsing the error message.

e.g. `e instanceof ThresholdAlertError`